src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Power-Strings.json

```diff
@@ -241,6 +241,13 @@
       "Reduce healing by %1$s%%."
     ]
   },
+  "stsjorbsmod:ShrapnelBloomPower": {
+    "NAME": "Shrapnel Bloom",
+    "DESCRIPTIONS": [
+      "This turn, your next non-legendary card is played %1d additional time, then #yexerts.",
```

Comment:
I don't think we can colorize just the "legendary" part of "non-legendary", but probably best to at least capitalize it like a keyword (and for consistency with the card text does)

Reply:
Fixed.